processing/src/test/java/io/druid/segment/data/BenchmarkIndexibleWrites.java

```diff
@@ -171,7 +171,7 @@ private void ensureCapacity(int capacity)
   private final Integer totalIndexSize = 1<<20;
 
   @BenchmarkOptions(warmupRounds = 100, benchmarkRounds = 100, clock = Clock.REAL_TIME, callgc = true)
-  @Test
+  //@Test
```

Comment:
can we add `@Ignore` instead of commenting out, this way it is still possible to run the tests explicitely